### DISCUSSION TITLE
nanoplot data value font size

### DISCUSSION THREAD

**Discussion by tpu-brunkhorst** [posit-dev/great-tables#413] [Q&A]
Great package!  Can I make the nanoplot data values have bigger font?  I especially like the non-interactive version for static tables.  I'm not seeing how to do that.  Thanks!

**Comment by blaylockbk** [comment]
I'm wondering the same thing. When I hover over a bar, I like that I can see the number, but the number is very small.
![image](https://github.com/user-attachments/assets/23b6d62f-5ebb-487b-8c4d-0199fee8a4d2)
